website/static/install.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+
+set -e
+
+if [ "$(uname -s)" = "Darwin" ] && [ "$(uname -m)" = "x86_64" ]; then
+    target="macos_x86_64"
+elif [ "$(uname -s)" = "Linux" ] && [ "$(uname -m)" = "x86_64" ]; then
+    target="linux_x86_64"
+elif [ "$(uname -s)" = "Linux" ] && [ "$(uname -m)" = "arm64" ]; then
+    target="linux_arm64"
+elif [ "$(uname -s)" = "Darwin" ] && [ "$(uname -m)" = "arm64" ]; then
+    target="macos_arm64"
+else
+    echo "Unsupported OS or architecture"
+    exit 1
+fi
+
+fetch() {
+    if which curl >/dev/null; then
+        if [ "$#" -eq 2 ]; then curl -L -o "$1" "$2"; else curl -sSL "$1"; fi
+    elif which wget >/dev/null; then
+        if [ "$#" -eq 2 ]; then wget -O "$1" "$2"; else wget -nv -O - "$1"; fi
+    else
+        echo "Can't find curl or wget, can't download package"
+        exit 1
+    fi
+}
+
+echo "Detected target: $target"
+
+url=$(
+    fetch https://api.github.com/repos/G-Research/fasttrackml/releases/latest |
+        tac | tac | grep -wo -m1 "https://.*$target.tar.gz" || true
+)
+if ! test "$url"; then
+    echo "Could not find release info"
+    exit 1
+fi
+
+echo "Downloading fml..."
+
+temp_dir=$(mktemp -dt fml.XXXXXX)
```

Comment:
possibly some content filters might block the script from download because of this name string. maybe just "fml.tmp"?

Reply:
the `X`'s are actually required by the GNU coreutils version of `mktemp`, so changing the _template_ to `fml.tmp` makes the script now fail on Linux

Reply:
also a content filter that looks at `XXXXXX` within a TLS stream and decides to block it would most likely block the download of our binary as well